learningequality/kolibri-design-system
# K tooltip widthconfigure

## Description
This PR configures the maxWidth of KTooltip,  If maxWidth is not passed it will maintain its current behavior of adjusting its size to fit its content

#### Issue addressed
#595

Addresses #*PR# HERE*

###  screenshots
If maxWidth is not passed, KTooltip will maintain its current behavior of adjusting its size to fit its content.
![before](https://github.com/learningequality/kolibri-design-system/assets/94701487/1539974c-e097-4883-81ab-7ee8b0a698e4)
If maxWidth passes.
![after](https://github.com/learningequality/kolibri-design-system/assets/94701487/e0d3778b-eabf-4f70-b898-3634fe6ce24a)


## Changelog



- [#PR no]
  - **Description:** This PR configures the maxWidth of KTooltip,  If maxWidth is not passed it will maintain its current behavior of adjusting its size to fit its content
  - **Addresses:**  [issue-595](https://github.com/learningequality/kolibri-design-system/issues/595)
  

[#PR no]: PR link

## Steps to test

1. Step 1 
  Run the KDS By  yarn dev
2. Step 2
Past this code snippet to the playground.vue file in following path  docs/pages /playground.vue

` <KIcon ref="icon" icon="info" />
    <KTooltip
      :maxWidth="'100px'"
      reference="icon"
      :refs="$refs"
    >
      Tooltip text tooltip text tooltip text tooltip text tooltip text
    </KTooltip> `

3. Past this in your browser   http://localhost:4000/playground  and see the changes

## (optional) Implementation notes

### At a high level, how did you implement this?
##### Briefly describe how this works 
If the maxWidth is passed like here
` <KIcon ref="icon" icon="info" />
    <KTooltip
      :maxWidth="'100px'"
      reference="icon"
      :refs="$refs"
    >
      Tooltip text tooltip text tooltip text tooltip text tooltip text
    </KTooltip> `
The maxWidth will get applied. but if maxWidth is not applied it will continue its current  behavior of adjusting its size to fit its content.

## Testing checklist



## Reviewer guidance




## After review


## Comments
